routes/api/v1/dashboards.js

```diff
@@ -0,0 +1,69 @@
+const Dashboard = require('../../../models/dashboards.js')
```

Comment:
Everywhere in this file, please insert a space between parents and params in function calls.